app/actions/DaemonActions.js

```diff
@@ -314,27 +314,50 @@ export const startWallet = (selectedWallet) => (dispatch, getState) => {
 };
 
 export const prepStartDaemon = () => (dispatch, getState) => {
+  console.log("prepstartdaemon");
   const { daemon: { daemonAdvanced, openForm, walletName } } = getState();
+  console.log(daemonAdvanced);
+  const cliOptions = ipcRenderer.sendSync("get-cli-options");
```

Comment:
Instead of having many `sendSync("get-cli-options")`, you can do it only once and set the cli-options into a selector.

Reply:
Yeah, was going to do that during final refactoring.